openmc/model/funcs.py

```diff
@@ -6,14 +6,19 @@
 from operator import attrgetter
 
 from openmc import (
-    XPlane, YPlane, Plane, ZCylinder, Quadric, Cylinder, XCylinder,
-    YCylinder, Material, Universe, Cell)
+    XPlane, YPlane, Plane, ZCylinder, Cylinder, XCylinder,
+    YCylinder, Universe, Cell)
 from openmc.checkvalue import (
     check_type, check_value, check_length, check_less_than,
     check_iterable_type)
 import openmc.data
 
 
+ZERO_CELSIUS_TO_KELVIN = 273.15
```

Comment:
Our physical constants are currently distributed throughout the Python API. What are your thoughts on creating a `constants.py` file to keep all of them in one place @paulromano?

I suppose it might be nice to limit what's in scope to only what you are using.. not sure about the tradeoff here...

Reply:
Yeah, I think that's generally a good idea. I didn't want to start moving around too much code in this one PR, but I think we ought to flag this for follow up, if you're ok with that.

Reply:
Yeah -- makes sense for that to be its own set of changes.